libs/wire-api/src/Wire/API/Routes/Public/Galley.hs

```diff
@@ -136,6 +136,13 @@ data Api routes = Api
              "size"
              (Range 1 500 Int32)
         :> Get '[Servant.JSON] (Public.ConversationList Public.Conversation),
+    listConversations ::
+      routes
+        :- Summary "Get all conversations (also returns remote conversations)"
+        :> ZUser
+        :> "list-conversations"
+        :> ReqBody '[Servant.JSON] Public.ListConversations
+        :> Post '[Servant.JSON] (Public.ConversationList Public.Conversation),
```

Comment:
I'd like to get a review on the endpoint naming here: Is `POST /list-conversations` a good name or would a different route be preferred?

Reply:
Before I make all the changes in nginz configs, perhaps I could get an early feedback on this question @akshaymankar ?

Reply:
This is consistent with `list-users`, `list-clients` and `list-prekeys`, so I like it :100: